languages/ar.ini

```diff
@@ -378,6 +377,7 @@ First = "الأول"
 First Name = "الاسم الأول"
 fix_metadata = "أجل، قم بإصلاح الميتاداتا؛سأنتظر"
 for search = "نتيجة بحث عن"
+for_search_html = "for search: '<strong>%%lookfor%%</strong>'"
```

Comment:
Also note that some of the other derived strings incorporate the English 'for search' text. So this may be a widespread problem affecting multiple strings in multiple files.